### ISSUE TITLE
Suggest archiving this repo

### ISSUE THREAD

**Issue by walles** [liato/android-bankdroid#692]
Latest change was in 2017, it's not on Google Play any more.

I vote for archiving this.

That said, it was a good ride, and except that it's not available any more the Bankdroid widget is still better than the Länsförsäkringar one (which is too large).

**Comment by goober** [comment]
I agree with you. Unfortunately I don't have permission to archive it. Maybe @liato is still around?
